File: mycollab-web/src/main/scala/com/mycollab/module/user/accountsettings/view/events/SettingEvent.scala
package com.mycollab.module.user.accountsettings.view.events

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd.
 * @since 5.0.5
 */
object SettingEvent {
    
    class GotoGeneralSetting(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

    class GotoTheme(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}
    
    class SaveTheme(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}
    
    class ResetTheme(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}
    
}

File: mycollab-web/src/main/java/com/mycollab/module/user/accountsettings/view/event/SettingEvent.kt
package com.mycollab.module.user.accountsettings.view.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
object SettingEvent {
    class GotoGeneralSetting(source: Any, val data: Any?) : ApplicationEvent(source)

    class GotoTheme(source: Any, val data: Any?) : ApplicationEvent(source)

    class SaveTheme(source: Any, val data: Any?) : ApplicationEvent(source)

    class ResetTheme(source: Any, val data: Any?) : ApplicationEvent(source)
}